Assets/MRTK/Core/Utilities/Gltf/Schema/GltfBufferView.cs

```diff
@@ -42,11 +43,31 @@ public class GltfBufferView : GltfChildOfRootProperty
         /// All valid values correspond to WebGL enums.
         /// When this is not provided, the bufferView contains animation or skin data.
         /// </summary>
-        public GltfBufferViewTarget target = GltfBufferViewTarget.None;
+        public GltfBufferViewTarget Target { get; set; }
+
+        [SerializeField]
+        private string target = null;
```

Comment:
Much of this is moved to the `ISerializationCallbackReceiver` handlers below. The default is still there.